src/hooks/useArticleLinksNavigation.js

```diff
@@ -101,6 +101,9 @@ export const useArticleLinksNavigation = (
 const makeLinkClickEvent = link => {
   const title = link.getAttribute('title')
   if (title) {
+    if (title.includes(':') && /^http(s?):\/\//.test(link.href)) {
```

Comment:
`link.href` is a property that uses the base path and returns a full URL.
If you want the value of the href attribute, you can use `link.getAttribute('href')`

It would probably work but it is fragile. It is possible for local links to have a full URL. A safer fix would be to check the hostname against a whitelist of wiki sites (wiktionary, wikiquote, etc) using something like that... with maybe special cases for common and wikidata
`
var url = new URL(link.href)
var site = url.hostname.match(/^(\w+)\.(\w+)\.org/)[2]
`

Reply:
perhaps, still the `link.getAttribute('href')` can cover most of the cases.

Reply:
regex `/^(\w+)\.(\w+)\.org/` should be enough

Reply:
Can you find the example of `It is possible for local links to have a full URL`?

Reply:
> Can you find the example of `It is possible for local links to have a full URL`?

It would be caused by bad wikitext. I don't know if I can search for an example.

The list you have below is not that bad. The title can have many forms but there is only 16 or so sister projects. Another option is just to check that `link.getAttribute('href') !== {lang}.wikipedia.org`

Reply:
Here’s complete list of the interwiki links and the shortcut 

https://en.wikipedia.org/wiki/Help:Interwiki_linking#Project_titles_and_shortcuts